server/go/api_create_new_static_route_service_impl.go

```diff
@@ -0,0 +1,39 @@
+/*
+ * Kusk Gateway API
+ *
+ * This is the Kusk Gateway Management API
+ *
+ * API version: 1.0.0
+ * Generated by: OpenAPI Generator (https://openapi-generator.tech)
+ */
+
+package openapi
+
+import (
+	"context"
+	"net/http"
+
+	kusk "github.com/GIT_USER_ID/GIT_REPO_ID/kusk"
+)
+
+// CreateNewStaticRouteApiService is a service that implements the logic for the CreateNewStaticRouteApiServicer
+// This service should implement the business logic for every endpoint for the CreateNewStaticRouteApi API.
+// Include any external packages or services that will be required by this service.
+type CreateNewStaticRouteApiService struct {
+	kuskClient kusk.Client
+}
+
+// NewCreateNewStaticRouteApiService creates a default api service
+func NewCreateNewStaticRouteApiService(kuskClient kusk.Client) CreateNewStaticRouteApiServicer {
+	return &CreateNewStaticRouteApiService{kuskClient: kuskClient}
+}
+
+// CreateStaticRoute - create new static route
+func (s *CreateNewStaticRouteApiService) CreateStaticRoute(ctx context.Context, staticRouteItem StaticRouteItem) (ImplResponse, error) {
+	staticRoute, err := s.kuskClient.CreateStaticRoute(staticRouteItem.Namespace, staticRouteItem.Name, staticRouteItem.EnvoyFleetNamespace, staticRouteItem.EnvoyFleetName)
+	if err != nil {
+		return Response(http.StatusInternalServerError, err), err
+	}
+	return Response(http.StatusOK, staticRoute), nil
```

Comment:
expected response for staticroutes POST is https://github.com/kubeshop/kuskgateway-api-server/blob/create_staticroute/api/openapi.yaml#L342 
Please transform CRD into API object

Reply:
Resolved.

Convert to `StaticRouteItem`.

Please mark as resolved, if you are happy with it.